documentation/modules/deploying/con-deploy-examples.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// deploying/assembly-deploy-tasks.adoc
+
+[id='deploy-examples-{context}']
+= Example configuration files
+
+[role="_abstract"]
+Use the example configuration files provided with Strimzi to deploy Kafka components and monitor your deployment.
+Example configuration files for custom resources contain key properties and values, which you can extend with additional supported configuration properties for your own deployment.
+
+== Example files location
+
+ifdef::Downloading[]
+The example files are provided with the downloadable release artifacts from {ReleaseDownload}.
+
+You can also access the example files directly from the
+link:https://github.com/strimzi/strimzi-kafka-operator/tree/{GithubVersion}/examples/[`examples` directory^].
+endif::Downloading[]
+
+ifndef::Downloading[]
+The example files are provided with the downloadable release artifacts from the {ZipDownload}.
+endif::Downloading[]
+
+You can download and apply the examples using the `kubectl` command-line tool.
+The examples can serve as a starting point when building your own Kafka component configuration for deployment.
+
+NOTE: If you installed Strimzi using the Operator, you can still download the example files and use them to upload configuration.
+
+== Example files provided with Strimzi
+
+The release artifacts include an `examples` directory that contains the configuration examples.
+
+.Examples directory
+[source]
+--
+examples
+├── user <1>
+├── topic <2>
+├── templates
+│   ├── topic-operator <3>
+│   └── cluster-operator <4>
+├── security <5>
+│   ├── tls-auth
+│   ├── scram-sha-512-auth
+│   └── keycloak-authorization
+├── mirror-maker <6>
+├── metrics <7>
+├── kafka <8>
+├── cruise-control <9>
+├── connect <10>
+└── bridge <11>
+--
+<1> `KafkaUser` custom resource configuration, which is managed by the User Operator.
+<2> `KafkaTopic` custom resource configuration, which is managed by Topic Operator.
+<3> Topic Operator template configuration for creating topics.
+<4> Cluster Operator template configuration for deploying Kafka components. Each supported Kafka component has a template configuration file. Includes templates for an ephemeral or persistent Kafka deployment.
```

Comment:
Sorry to ask a horrid question.

What is the difference between `examples/templates/connect-template.yaml` and `examples/connect/kafka-connect.yaml`?

Reply:
Two different approaches. As far as I understand, OpenShift templates are like installing with Helm charts.
Is that right, @scholzj?

Reply:
Well, Helm is a way more powerful than the templates are. The OpenShift templates really just replace some keys with specified values. Maybe you should just not mention the templates at all and we should delete them, as the templates are crap and are anyway not really supported in OCP 4 that well.

Reply:
Thanks, @scholzj. One for the next sync up? Maybe we keep these callouts in for completeness until a decision is made?

Reply:
As discussed, the _templates_ have been removed.